test/integ/routers/alpha-router/alpha-router.integration.test.ts

```diff
@@ -844,6 +922,134 @@ describe('alpha router integration', () => {
       });
     });
   }
+
+  describe('QuoterV3', () => {
+    const WISE_MAINNET = new Token(
+      1,
+      '0x66a0f676479Cee1d7373f3DC2e2952778BfF5bd6',
+      18,
+      'WISE',
+      'WISE'
+    );
+    const tradeType = TradeType.EXACT_INPUT;
+
+    const TRIBE_MAINNET = new Token(
+      1,
+      '0xc7283b66Eb1EB5FB86327f08e1B5816b0720212B',
+      18,
+      'TRIBE',
+      'TRIBE'
+    );
+
+    const BOND_MAINNET = new Token(
+      1,
+      '0x0391D2021f89DC339F60Fff84546EA23E337750f',
+      18,
+      'BOND',
+      'BOND'
+    );
+
+    const SOCKS_MAINNET = new Token(
+      1,
+      '0x23B608675a2B2fB1890d3ABBd85c5775c51691d5',
+      18,
+      'SOCKS',
+      'SOCKS'
+    );
+
+    const APE_MAINNET = new Token(
+      1,
+      '0x4d224452801aced8b2f0aebe155379bb5d594381',
+      18,
+      'APE',
+      'APE'
+    );
+
+    const STETH_MAINNET = new Token(
+      1,
+      '0xae7ab96520de3a18e5e111b5eaab095312d7fe84',
+      18,
+      'STETH',
+      'STETH'
+    );
+
+    beforeAll(async () => {
+      await hardhat.fund(
+        alice._address,
+        [parseAmount('1000', WISE_MAINNET)],
+        [
+          '0xa66e19298a50d6e33fd6756af15d81dae39b47ee', // WISE token
+        ]
+      );
+      const aliceWISEBalance = await hardhat.getBalance(
+        alice._address,
+        WISE_MAINNET
+      );
+      expect(aliceWISEBalance).toEqual(parseAmount('1000', WISE_MAINNET));
+
+      await hardhat.fund(
+        alice._address,
+        [parseAmount('10000', TRIBE_MAINNET)],
+        [
+          '0xea7b32c902daff20bda7b9d7b0964ff0cd33d7ea', // TRIBE whale
+        ]
+      );
+      const aliceTRIBEBalance = await hardhat.getBalance(
+        alice._address,
+        TRIBE_MAINNET
+      );
+      expect(aliceTRIBEBalance).toEqual(parseAmount('10000', TRIBE_MAINNET));
+    });
+
+    describe(`${
+      tradeType === TradeType.EXACT_INPUT ? 'exactInput' : 'exactOutput'
+    } mixedPath routes`, () => {
+      describe('+ simulate swap', () => {
+        it('WISE -> USDC', async () => {
```

Comment:
Those were just for tests I was running locally since I have to create `Token` objects for each token I wanted to test with. I'll remove before merge and clean up the integ tests file